Lib9c/Action/ClaimStakeReward.cs

```diff
@@ -37,6 +37,7 @@ public override IAccountStateDelta Execute(IActionContext context)
             }
 
             var states = context.PreviousStates;
+            CheckObsolete(ClaimStakeReward3.HardForkIndex, context);
```

Comment:
If use `ClaimStakeReward3.HardForkIndex` as is, `ClaimStakeReward3` and `ClaimStakeReward`("xxx2") available in the block which index is `ClaimStakeReward3.HardForkIndex`.

```suggestion
            CheckObsolete(ClaimStakeReward3.HardForkIndex - 1, context);
```